code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -85,7 +86,7 @@
 		..()
 		new /obj/item/weapon/screwdriver(src)
 		new /obj/item/weapon/wrench(src)
-		new /obj/item/weapon/weldingtool(src)
+		new /obj/item/weapon/weldingtool/largetank(src)
 		new /obj/item/weapon/crowbar(src)
 		new /obj/item/weapon/wirecutters(src)

```

Comment:
Since you're already modifying this, may want to make sure the wirecutters are red and the crowbar is red, too....because Syndiessss.